### PR TITLE
2595 download all agents logs in preview pane

### PR DESCRIPTION
**Pull request by ilija-lazoroski**
# What does this PR do?

Fixes part of #2595 .

Add any further explanations here.

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [x] Was the documentation framework updated to reflect the changes?
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [x] Added relevant unit tests?
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by running twice and downloading two files.
* [x] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: monkey/monkey_island/cc/ui/src/components/ui-components/LogDownloadButtons.tsx [hunk 1]
```diff
@@ -11,11 +11,17 @@ type Props = { url: string, filename: string, variant?: string }
 const LOG_FILE_NOT_FOUND_ERROR = "The server returned a 404 (NOT FOUND) response: " +
                                  "The requested log files do not exist."
 
-export const AgentLogDownloadButton = ({ url, filename, variant = 'primary' }: Props) => {
+export const AgentLogDownloadButton = ({ url, agentIds, filename, variant = 'primary' }: Props) => {
```
Comment (shreyamalviya) [inline monkey/monkey_island/cc/ui/src/components/ui-components/LogDownloadButtons.tsx:14]: Update line 9 with `agentIds`.

**Review by shreyamalviya — CHANGES_REQUESTED** [review]
We should talk about how we want to handle the error modal for this case. It'll pop up even if one file is missing, but it gives no information about which file was missing or exactly how many files were missing.

File: monkey/monkey_island/cc/ui/src/components/ui-components/LogDownloadButtons.tsx (hunk 1 above)
Comment (ilija-lazoroski) [reply monkey/monkey_island/cc/ui/src/components/ui-components/LogDownloadButtons.tsx:14]: That is going to be fixed in the other PR.

**Comment by ilija-lazoroski** [comment]
> > We should talk about how we want to handle the error modal for this case. It'll pop up even if one file is missing, but it gives no information about which file was missing or exactly how many files were missing.
> 
> Agreed. It should be specific about which agents are missing logs.

@mssalvatore @shreyamalviya I agree with the above. Can we fix that in another PR?

File: monkey/monkey_island/cc/ui/src/components/ui-components/LogDownloadButtons.tsx (hunk 1 above)
Comment (ilija-lazoroski) [reply monkey/monkey_island/cc/ui/src/components/ui-components/LogDownloadButtons.tsx:14]: https://github.com/guardicore/monkey/pull/2704

**Review by mssalvatore — APPROVED** [review]

**Pull request merged**